File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/PaymentResponse.java
package com.cinema.ticket_booking.data.model.response;
import com.google.gson.annotations.SerializedName;
public class PaymentResponse {
    @SerializedName("id")          private String id;
    @SerializedName("bookingId")   private String bookingId;
    @SerializedName("bookingCode") private String bookingCode;
    @SerializedName("amount")      private double amount;
    @SerializedName("status")      private String status;
    @SerializedName("paymentUrl")  private String paymentUrl;
    @SerializedName("paidAt")      private String paidAt;
    public String getId()          { return id; }
    public String getBookingId()   { return bookingId; }
    public double getAmount()      { return amount; }
    public String getStatus()      { return status; }
    public String getPaymentUrl()  { return paymentUrl; }
    public String getPaidAt()      { return paidAt; }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/PaymentResponse.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class PaymentResponse(
    @SerializedName("id")
    val id: String?,

    @SerializedName("bookingId")
    val bookingId: String?,

    @SerializedName("bookingCode")
    val bookingCode: String?,

    @SerializedName("amount")
    val amount: Double,

    @SerializedName("status")
    val status: String?,

    @SerializedName("paymentUrl")
    val paymentUrl: String?,

    @SerializedName("paidAt")
    val paidAt: String?,

    @SerializedName("pointsUsed")
    val pointsUsed: Long?,

    @SerializedName("pointDiscount")
    val pointDiscount: Double?,

    @SerializedName("remainingAmount")
    val remainingAmount: Double?
)
